src/templates/android.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
 const { create, convert } = require('xmlbuilder2');
 const fs = require('fs-extra');
-const { formatName } = require('../utils');
+const { formatName, ensureAndoridFiles } = require('../utils');
```

Comment:
Spelling, should be ensureAndroidFiles

Reply:
We made an off-github review and will actually delete this function instead 👍